WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -20117,6 +20138,7 @@
 				FAB8AB5F25AFFD0600F9F8A0 /* JetpackRestoreStatusCoordinator.swift in Sources */,
 				F913BB0E24B3C58B00C19032 /* EventLoggingDelegate.swift in Sources */,
 				4089C51022371B120031CE78 /* TodayStatsRecordValue+CoreDataClass.swift in Sources */,
+				3F8B45A8292C1F2500730FA4 /* MigrationSuccesCardView.swift in Sources */,
```

Comment:
This is a non-blocking nit since it doesn't affect compilation or logic, but `Success` is misspelled.